src/hotspot/share/memory/universe.cpp

```diff
@@ -122,7 +122,6 @@ LatestMethodCache* Universe::_throw_illegal_access_error_cache = NULL;
 LatestMethodCache* Universe::_throw_no_such_method_error_cache = NULL;
 LatestMethodCache* Universe::_do_stack_walk_cache     = NULL;
 
-bool Universe::_verify_in_progress                    = false;
```

Comment:
This cleanup seems completely unrelated to your mutex change and is best left to a separate cleanup RFE.

Reply:
Done.